views/js/qtiCreator/helper/itemLoader.js

```diff
@@ -63,12 +63,14 @@ define([
                 Promise.all([langList, itemRdf]).then(([languagesList, data]) => {
                     var loader, itemData, newItem;
 
-                    let newObject = {};
-                    for (const response in data.itemData.responses) {
-                        for (const mapKey in data.itemData.responses[response].mapping) {
-                            newObject[decodeHtml(mapKey)] = data.itemData.responses[response].mapping[mapKey];
+                    if (data.itemData) {
+                        let newObject = {};
+                        for (const response in data.itemData.responses) {
```

Comment:
if `data.itemData` is null, throws with `TypeError: Cannot read property 'responses' of null`